nextest-runner/default-config.toml

```diff
@@ -55,6 +55,9 @@ fail-fast = true
 # Treat a test that takes longer than this as slow, and print a message.
 slow-timeout = "60s"
 
+# Number of slow-timeout periods to tolerate before terminating a test.
+slow-timeout-tolerance = 2
```

Comment:
What I'd do here is something like:

```
slow-timeout = { display-after = "60s", kill-after = 2 }
```

Define `kill-after = 0` as the default (never kill).

Also, maintain the current compatibility with strings by using a custom deserialize impl. See https://github.com/nextest-rs/nextest/compare/main...sunshowers:slow-timeout for an example of how to do that (feel free to use that code as you wish).

Reply:
thanks for providing readily available code for this feature! i cherry-picked your commit. however, i removed the Default/Custom distinction as it seemed redundant. please let me know if i missed something that made that distinction necessary that got lost with my changes.